test/performance/array/distCreate.ml-compopts

```diff
@@ -0,0 +1 @@
+-scorrectness=false
```

Comment:
Does correctness need to be a param? Might be a little cleaner as a const, and then throw this in the ml-execopts

Reply:
This is also a nit, but I tend to have the default mode be the one I'm most interested in if I'm running this test manually i.e. for things designed as performance tests I tend to have the performance mode be the default (though I'm not 100% consistent with that.)